### DISCUSSION TITLE
Release v1.0

### DISCUSSION THREAD

**Discussion by jarabroz** [Dlubal-Software/Dlubal_CSharp_Client#2] [Announcements]
Initial release of the library
Dlls are in zip folder DlubalC.Library.zip

<hr /><em>This discussion was created from the release <a href='https://github.com/Dlubal-Software/Dlubal_CSharp_Client/releases/tag/1.0.0'>Release v1.0</a>.</em>

**Comment by michaelvantelgen-work** [comment]
Hi @jarabroz , an example page is mentioned in the readme but that link doesn't seem to exist:
https://github.com/Dlubal-Software/Dlubal_CSharp_Client/blob/main/examples

Would be helpful to get some materials to get started :)

Especially would like some info on how to connect to the model.

Thank you! Looks great by the way.

Michael van Telgen

**Reply by jarabroz** [reply]
Hi @michaelvantelgen-Arcadis,
I have added some examples. As start Cantilever is good start 😉 

Jaroslav

**Reply by michaelvantelgen-work** [reply]
Hi @jarabroz,

Upon running the cantilever tool example I get an error message that reads:

> "System.ServiceModel.CommunicationException: No connection could be made because the target machine actively refused it. (localhost:8081)"

Could this be due to a firewall? I'll ask our IT service desk too.

Thanks,
Michael

**Reply by jarabroz** [reply]
Hi @michaelvantelgen-Arcadis ,

you need to check option Web Services - Start server automatically with the application in Program Options & Settings. See picture.
![image](https://user-images.githubusercontent.com/29481431/143880293-141586e2-e2c5-4676-8b2f-5581c1dabde4.png)
Once you enable it and restart application then you should see dialogue from firewall - firewall is asking you to allow this connection. You should be able to allow it.

**Reply by michaelvantelgen-work** [reply]
@jarabroz, thanks! That solved the issue. I was able to model the cantilever using the provided code.

**Reply by michaelvantelgen-work** [reply]
@jarabroz I have another question. My class library project targets .NET Framework 4.8 but I believe the RfemSoapWsCoreClient targets .NET 5. I'm getting error messages such as:

> System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.'

I think it's due to the difference in target framework but I'm not sure. When I run exactly the same code but in a .NET 5 class library it works fine.

Do you have experience with this?

Thanks again!

**Reply by jarabroz** [reply]
Yes, we have .NET 5 Core. It could be the difference in target framework. You can change settings of https://github.com/Dlubal-Software/Dlubal_CSharp_Client/tree/main/source_code/RfemSoapWsCoreClient to .NET Framework 4.8. But it needs to also change some references of Microsoft libraries. I recommend to use .NET 5.0
